qiskit/opflow/gradients/natural_gradient.py

```diff
@@ -363,8 +400,8 @@ def _lasso(
         <https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.Lasso.html>`
 
         Args:
-            a: mxn matrix
-            c: m vector
+            metric: mxn matrix
+            gradient: m vector
```

Comment:
I would give more explanation in docs. Also m vector does not sound right.
```suggestion
            metric: Matrix of size mxn.
            gradient: Vector of size m.
```